openapi.yaml

```diff
@@ -14723,6 +14733,14 @@ components:
                   type: string
                   x-stoplight:
                     id: iusxfgcmg72tl
+                azure_tenant_id:
+                  type: string
+                  x-stoplight:
+                    id: obk7v9hlx3jv4
+                azure_subscription_id:
+                  type: string
+                  x-stoplight:
+                    id: nrvn3j8xq81da
```

Comment:
question: if they are the creds of clients azure account, shouldn't we hide the sensitive info as for others ? i.e we anszer the access key but not the secret key for aws cr